landa/organization_management/doctype/landa_member/landa_member.js

```diff
@@ -54,5 +56,16 @@ frappe.ui.form.on("LANDA Member", {
 					});
 			});
 		}
-	},
+		frm.add_custom_button(__('Linking as Contact Person'), function() {
+			 frappe.call({
+                method: 'landa.organization_management.doctype.landa_member.link_contact_person',
```

Comment:
```suggestion
                method: 'landa.organization_management.doctype.landa_member.landa_member.link_contact_person',
```

So müsste es funktionieren.

```
/app /modul                  /ordner /ordner      /datei        /funktion
landa.organization_management.doctype.landa_member.landa_member.link_contact_person
```